StardewArchipelago/Stardew/StardewItemManager.cs

```diff
@@ -519,9 +519,9 @@ private void AddObjectByColor(ObjectData objectData, StardewObject stardewObject
                 return; //Its an object with no tags attached, so wouldn't have a color.
             }
             var firstColor = objectData.ContextTags.FirstOrDefault(x => x.Contains("color_"));
-            if (firstColor is null)
+            if (firstColor is null || Constants.Modded.IgnoredModdedStrings.JojaRouteSVE.Contains(stardewObject.Name))

```

Comment:
I think the items should make it to the list, and get filtered upon usage instead.
StardewItemManager should, as much as possible, hold all the data available, not a filtered version of it

Reply:
Plus, I'm not a fan of introducing mod-related code in that file